package.json

```diff
@@ -29,8 +29,9 @@
       "lockFileMaintenance": {
         "enabled": false
       },
-      "rebaseStalePrs": true,
-      "semanticCommits": true,
+      "semanticCommits": false,
+      "prTitle": "{{semanticCommitType}}({{semanticCommitScope}}): {{#if isPin}}Pin{{else}}{{#if isRollback}}Roll back{{else}}Update{{/if}}{{/if}} {{depName}} to {{#unless isRange}}v{{/unless}}{{#if isMajor}}{{newVersionMajor}}.x{{else}}{{newVersion}}{{/if}}",
```

Comment:
😅👍

Reply:
🤕 😂

Reply:
Just double checked this—love that this is scoped to *all* packages, not just SEEK ones. We should probably call this out in the readme as one of this package's selling points.

Reply:
Added to the readme 👍